tests/integration_tests/test_model_training_options.py

```diff
@@ -170,8 +170,8 @@ def test_model_save_resume(generated_data, tmp_path):
     model_definition = {
         'input_features': input_features,
         'output_features': output_features,
-        'combiner': {'type': 'concat', 'fc_size': 14},
```

Comment:
I'm guessing the reason you are removing the fc layers from the combiner is because you are trying to make it work only for the numerical features ate the moment and then you'll look into the other features / combiners, right?

Reply:
Yes...that is correct.  I'm trying to get this to work with a single numeric input feature and a single numeric output feature with only a fully connected stack on the output feature.  I figure is a simple model that will cover 90% of the requirements.  Once we can get this to work, we can try more complicated input/output features and neural network structures.